README.md

```diff
@@ -91,3 +91,32 @@ test framework.
 
 If you would like to develop and update the go binaries you will need to install 
 [go 1.12](https://golang.org/doc/install#install) and [upx](https://upx.github.io/)
+
+## Proxy Issues
+
+If your builds are not completing and have errors you may need to examine your build environment for `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables. A few examples of build output that may indicate issues with these values are below.
+
+```
+// ... 
+-----> Installing binaries
+       engines.node (package.json):  10
+       engines.npm (package.json):   unspecified (use default)
+
+       Resolving node version 10...
+       Error: Unknown error installing "10" of node
```

Comment:
We could detect this failure, and the `HTTP_PROXY` env var, and give a better help message